experimental/framework/tinylicious-client/src/test/TinyliciousClient.spec.ts

```diff
@@ -9,81 +9,67 @@ import { DiceRoller } from "@fluid-example/diceroller";
 import { SharedMap, SharedDirectory } from "@fluid-experimental/fluid-framework";
 import { ContainerSchema } from "@fluid-experimental/fluid-static";
 import {
+    TinyliciousClient,
     TinyliciousConnectionConfig,
     TinyliciousContainerConfig,
 } from "..";
-import { ResettableTinyliciousClient } from "./ResettableTinyliciousClient";
 
-describe("TinyliciousClient pre-initialization", () => {
+describe("TinyliciousClient", () => {
+    let tinyliciousClient: TinyliciousClient;
     let documentId: string;
     beforeEach(() => {
         documentId = uuid();
+        tinyliciousClient = new TinyliciousClient();
     });
 
     /**
-     * Scenario: test if TinyliciousClient can get a container without being initialized.
+     * Scenario: test if TinyliciousClient can be instantiated without a port
+     * number specified.
      *
-     * Expected behavior: an error should be thrown when trying to get a container
-     * without initializing TinyliciousClient.
+     * Expected behavior: an error should not be thrown nor should a rejected promise
+     * be returned.
      */
-    it("cannot get a container without initializing", async () => {
+    it("can create instance without specifying port number", async () => {
         const containerConfig: TinyliciousContainerConfig = { id: documentId };
         const schema: ContainerSchema = {
             name: documentId,
             initialObjects: {
                 map1: SharedMap,
             },
         };
-        const containerAndServices = ResettableTinyliciousClient.getContainer(containerConfig, schema);
-        const errorFn = (error) => {
-            assert.notStrictEqual(error.message, undefined, "TinyliciousClient error is undefined");
-            return true;
-        };
+        const containerAndServices = tinyliciousClient.createContainer(containerConfig, schema);
 
-        await assert.rejects(
+        await assert.doesNotReject(
             containerAndServices,
-            errorFn,
-            "TinyliciousClient can get a container without initialization",
+            () => {},
```

Comment:
I'm not super familiar with the error validation callback, but based on some experimentation it looks like a callback which doesn't return `true` will suppress your custom message.  I think you can just use undefined here since there's no additional validation needed.

```suggestion
            undefined,
```

Reply:
good catch! But I don't think `undefined` is an acceptable argument type for the `error` parameter, so I'm just going to remove it so the custom message is unblocked.

Reply:
`undefined` should be fine [based on this documentation](https://nodejs.org/api/assert.html#:~:text=If%20the%20error%20is%20of%20a%20different%20type,caller.).  Although it works, best practice is to avoid a string as a second argument [based on this documentation](https://nodejs.org/api/assert.html#:~:text=Due%20to%20the%20confusing,argument.).

Reply:
I was getting compilation errors when setting the `error` parameter as `undefined`. Looking into it further, it looks like the error argument is of type `AssertPredicate`, which has the following type `type AssertPredicate = RegExp | (new () => object) | ((thrown: any) => boolean) | object | Error;`. So as discussed, I changed the value to `() => true` instead to prevent message suppression.